packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -216,6 +229,17 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
   };
 }
 
+/**
+ * Normalize name or options to `BindingFromClassOptions`
+ * @param nameOrOptions - Name or options for binding from class
+ */
+function toOptions(nameOrOptions?: string | BindingFromClassOptions) {
```

Comment:
ditto

Reply:
They are in two packages. It's not worth the effort IMO.